vault/logical_system.go

```diff
@@ -72,14 +72,13 @@ func NewSystemBackend(core *Core, logger log.Logger) *SystemBackend {
 	db, _ := memdb.NewMemDB(systemBackendMemDBSchema())
 
 	b := &SystemBackend{
-		Core:      core,
-		db:        db,
-		logger:    logger,
-		mfaLogger: core.baseLogger.Named("mfa"),
-		mfaLock:   &sync.RWMutex{},
+		Core:       core,
+		db:         db,
+		logger:     logger,
+		mfaBackend: NewPolicyMFABackend(core, logger),
```

Comment:
`NewPolicyMFABackend` needs to be defined in OSS as well. Currently, this is only defined in ENT in policy_mfa_ent.go
Also, not sure if `NewSystemBackend` should even configure mfaBackend this way. Maybe, configuring the mfa part should be done in an ENT specific module?